docs/fiddles/windows/manage-windows/new-window/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+</head>
+<body>
+  <h2>Create a new window</h2>
+  <h3>Supports: Win, MacOS, Linux <span>|</span> Process: Main</h3>
+  <button id="new-window">View Demo</button>
+  <p>The <code>BrowserWindow</code> module gives you the ability to create new windows in your app. This main process module can be used from the renderer process with the <code>remote</code> module, as is shown in this demo.</p>
+  <p>There are a lot of options when creating a new window. A few are in this demo, but visit the <a id="browser-window-link" href="">documentation<span>(opens in new window)</span></a>
```

Comment:
Could you hook `shell.openExternal()` here to open the docs in a new window rather than the Electron example?

Reply:
Sure , i already have an event listener that opens the browser window example, i guess i can switch it for the docs and have it implemented inside of the index.html file, thanks